src/main/java/com/minecolonies/coremod/colony/buildings/BuildingFarmer.java

```diff
@@ -307,6 +307,13 @@ public void onDestroyed()
         }
     }
 
+    @Override
+    public void onWakeUp()
+    {
+       resetFields();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)